server/data/licenceClient.ts

```diff
@@ -230,6 +239,16 @@ export class LicenceClient {
     await db.query(query)
   }
 
+  async setLicenceInCvl(licence_in_cvl: boolean, bookingId: number): Promise<void> {
+    await db.inTransaction(async (client) => {
+      await client.query({
+        text: 'UPDATE v_licences_excluding_deleted SET licence_in_cvl = $1 where booking_id = $2',
+        values: [licence_in_cvl, bookingId],
+      })
+      return setLicenceVersionInCvl(licence_in_cvl, bookingId)
```

Comment:
mmh.. think the client is the transactional bit - so needs to be passed into the setLicenceVersionInCvl to continue the transaction?

(wonder this is wrong elsewhere)

Reply:
probably need to look at this deeper...

Reply:
```
export const inTransaction = async (callback) => {
  const client = await pool.connect()
  try {
    await client.query('BEGIN')
    const result = await callback(client)
    await client.query('COMMIT')
    return result
  } catch (error) {
    await client.query('ROLLBACK')
    throw error
  } finally {
    client.release()
  }
}
```

yes - I think they all need to be done on the same client instance